src/Utility/LogStream.C

```diff
@@ -62,8 +62,10 @@ void utl::LogStream::addExtraLog(std::shared_ptr<std::ostream> extra, bool clear
 int utl::LogStream::precision(int streamsize)
 {
   int result = streamsize;
-  if (m_out)
-    result = m_out->precision(streamsize);
+  if (m_out) {
+    result = m_out->precision();
```

Comment:
@kmokstad turns out this was the root of the troubles i mentioned on skype. this is way less intrusive than the alternative i 'presented' there.